vnext/Shared/OInstance.cpp

```diff
@@ -430,6 +435,23 @@ InstanceImpl::InstanceImpl(
     if (m_devSettings->jsiRuntimeHolder) {
```

Comment:
>(m_devSettings->jsiRuntimeHolder [](start = 7, length = 32)

Should we add an assert to check that this is set only when m_devSettings->jsiEngineOverride == JSIEngineOverride::Default?